content/blog/2019/root-transition/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: User guide for extenting Istio root certificate lifetime
+description: The user guide to extend Istio root certificate lifetime.
+publishdate: 2019-06-06
+subtitle:
+attribution: Oliver Liu
+twitter:
+keywords: [security, PKI, certificate, Citadel]
+
+---
+
+## Background
+
+Istio self-signed certificates have the default lifetime of 1 year.
```

Comment:
can we mention which releases?  I assume this doesn't apply to 1.2 or 1.1.8.

Reply:
Good point. But 1.2 is not officially out yet, right?

Reply:
Good point. I will add.